src/LibChorus/merge/xml/generic/ChorusNotesMergeEventListener.cs

```diff
@@ -18,7 +18,9 @@ public class ChorusNotesMergeEventListener : IMergeEventListener, IDisposable
 		private FileStream _readerStream;
 		private TempFile _tempFile;
 		private string _path;
+		[Obsolete("Use TimeFormatWithTimeZone instead, as TimeFormatNoTimeZone produces incorrect results when used with DateTime.Now")]
 		static public string TimeFormatNoTimeZone = "yyyy-MM-ddTHH:mm:ssZ";
+		static public string TimeFormatWithTimeZone = "yyyy-MM-ddTHH:mm:ssK";
```

Comment:
This also is unused outside of Chorus, and while we are at it we could unify all of these into one place.

Reply:
I've been looking through the code and haven't found a totally obvious place to unify these. Put them in LiftUtils? Then it looks strange for the Conflict code to be using `LiftUtils.TimeFormatWithTimeZone`. Make a new class called TimeUtils? Then the patch becomes larger and the chance of conflicts when I cherry-pick it into other branches becomes larger. I'm trying to make a minimal bugfix patch; any refactoring I'd prefer to do in a separate PR after the bugfix goes through.